<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Entree</class>
 <widget class="QDialog" name="Entree">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>345</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Monaco</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Nouvelle Entree</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/box--plus.png</normaloff>../icons/box--plus.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_4">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>110</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>110</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 170, 255);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="text">
      <string>Nouvelle Entree</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>79</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>79</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Code</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditCode">
       <property name="minimumSize">
        <size>
         <width>358</width>
         <height>30</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>79</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Quantit√©</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="spinBoxQte">
       <property name="minimumSize">
        <size>
         <width>358</width>
         <height>30</height>
        </size>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>85</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>79</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Prix</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QSpinBox" name="spinBoxPrix">
       <property name="minimumSize">
        <size>
         <width>358</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QPushButton" name="pushButtonAddEntree">
       <property name="minimumSize">
        <size>
         <width>101</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>Entree</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QPushButton" name="pushButtonClose">
       <property name="minimumSize">
        <size>
         <width>101</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDateEdit" name="dateEditDate">
       <property name="currentSection">
        <enum>QDateTimeEdit::DaySection</enum>
       </property>
       <property name="displayFormat">
        <string>dd/MM/yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>Entree</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
